### PR TITLE
Add migration scripts for user store migration

### PR DESCRIPTION
**Pull request by sajinieKavindya**
Add migration scripts for user store migration.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md [hunk 1]
```diff
@@ -17,7 +17,21 @@ This guide explains the recommended strategy for upgrading from the Micro Integr
...
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with the Micro Integrator of EI 7.0.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file. 
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:26]:
```suggestion
There are changes in the database structure (schema) that is used in EI 7.0.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md [hunk 2]
```diff
@@ -17,7 +17,21 @@ This guide explains the recommended strategy for upgrading from the Micro Integr
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with the Micro Integrator of EI 7.0.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file. 
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:22]:
```suggestion
If you are using a JDBC user store with the Micro Integrator of EI 7.0.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md [hunk 3]
```diff
@@ -17,7 +17,21 @@ This guide explains the recommended strategy for upgrading from the Micro Integr
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with the Micro Integrator of EI 7.0.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file. 
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:20]:
```suggestion
If you are using LDAP user store with the Micro Integrator of EI 7.0.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md [hunk 4]
```diff
@@ -27,7 +27,21 @@ Follow the instructions below to start the upgrade!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.6.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:30]:
```suggestion
If you are using LDAP user store with the Micro Integrator of EI 6.6.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md [hunk 5]
```diff
@@ -27,7 +27,21 @@ Follow the instructions below to start the upgrade!
...
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:32]:
```suggestion
If you are using a JDBC user store with the Micro Integrator of EI 6.6.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md [hunk 6]
```diff
@@ -27,7 +27,21 @@ Follow the instructions below to start the upgrade!
...
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:36]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.6.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md [hunk 7]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.5.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.5.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md [hunk 8]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.5.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.5.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md [hunk 9]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.5.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:33]:
```suggestion
If you are using LDAP user store with EI 6.5.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md [hunk 10]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.4.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.4.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md [hunk 11]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.4.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.4.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md [hunk 12]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.4.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.4.x.md:33]:
```suggestion
If you are using LDAP user store with EI 6.4.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md [hunk 13]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.3.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md:33]:
```suggestion
If you are using LDAP user store with EI 6.3.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md [hunk 14]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.3.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md [hunk 15]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.3.x.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.3.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md [hunk 16]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.2.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md:33]:
```suggestion
If you are using LDAP user store with EI 6.2.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md [hunk 17]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.2.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md [hunk 18]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.2.x.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.2.0. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md [hunk 19]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.1.1, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md:33]:
```suggestion
If you are using LDAP user store with EI 6.1.1, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md [hunk 20]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.1.1, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md [hunk 21]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.1.1. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md [hunk 22]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
...
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.1.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
+
+#### Update the database structure
+
+There are changes in the database structure (schema) that is used in EI 7.1.0. To update the database schema:
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:39]:
```suggestion
There are changes in the database structure (schema) that is used in EI 6.1.1. To update the database schema:
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md [hunk 23]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.1.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
+
+If you are using a JDBC user store with the Micro Integrator of EI 7.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:35]:
```suggestion
If you are using a JDBC user store with EI 6.1.0, you can use the same databases with APIM 4.0.0. However, you need to apply the following updates.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md [hunk 24]
```diff
@@ -30,7 +30,21 @@ Follow the instructions below to start the migration!
 
 ### Migrating the user store
 
-If you are already using a JDBC or LDAP user store with EI 6.1.0, you can simply connect the same to the Micro Integrator.
+If you are using LDAP user store with the Micro Integrator of EI 7.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:33]:
```suggestion
If you are using LDAP user store with EI 6.1.0, you can simply connect the same to the Micro Integrator of API-M 4.0.0 by updating the configuration details in the Micro Integrator's `deployment.toml` file.
```

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md (hunk 1 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:26]: Done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md (hunk 2 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:22]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md (hunk 3 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-7.0.x.md:20]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md (hunk 4 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:30]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md (hunk 5 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:32]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md (hunk 6 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.6.x.md:36]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md (hunk 7 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:39]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md (hunk 8 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:35]: done

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md (hunk 9 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.5.x.md:33]: done

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**